dipy/workflows/tracking.py

```diff
@@ -88,18 +84,20 @@ def _core_run(self, stopping_path, stopping_thr, seeding_path,
                 affine=affine)
         logging.info('seeds done')
 
-        streamlines = LocalTracking(direction_getter, classifier,
-                                    seeds, affine, step_size=.5)
+        streamlines_generator = LocalTracking(direction_getter, classifier,
+                                              seeds, affine,
+                                              step_size=step_size)
         logging.info('LocalTracking initiated')
 
-        tractogram = Tractogram(streamlines, affine_to_rasmm=np.eye(4))
+        tractogram = Tractogram(streamlines_generator,
+                                affine_to_rasmm=np.eye(4))
         save(tractogram, out_tract)
-
         logging.info('Saved {0}'.format(out_tract))
 
     def run(self, pam_files, stopping_files, seeding_files,
             stopping_thr=0.2,
             seed_density=1,
+            step_size=0.5,
             tracking_method="deterministic",
```

Comment:
Can you replace "deterministic" by "eudx"

Reply:
done. thx.